### ISSUE TITLE
LUA error when hovering over the icon in Titan Panel (right side)

### ISSUE THREAD

**Issue by raysmith59** [SavedInstances/SavedInstances#214]
Latest version, haven't seen it before.

2x SavedInstances\SavedInstances-8.0.9.lua:3875: attempt to index field '?' (a nil value)
SavedInstances\SavedInstances-8.0.9.lua:3875: in function `ShowTooltip'
SavedInstances\SavedInstances-8.0.9.lua:2427: in function <SavedInstances\SavedInstances.lua:2425>
Titan\LDBToTitan.lua:327: in function <Titan\LDBToTitan.lua:286>

Locals:
self = <table> {
 SetDefaultModuleLibraries = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:398
 localarr#categoryrow = <table> {
 }
 lastrefreshlock = 431778.486000
 EnableModule = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:363
 HeaderFont = <function> defined @SavedInstances\SavedInstances.lua:3325
 CancelTimer = <function> defined @DataStore\libs\AceTimer-3.0\AceTimer-3.0.lua:147
 RequestLockInfo = <function> defined @SavedInstances\SavedInstances.lua:2522
 localarr#blankrow = <table> {
 }
 TRADE_SKILL_LIST_UPDATE = <function> defined @SavedInstances\SavedInstances.lua:4376
 SetDefaultModulePrototype = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:440
 defaultModuleState = true
 EncounterEnd = <function> defined @SavedInstances\SavedInstances.lua:2625
 localarr#columns = <table> {
 }
 ScheduleTimer = <function> defined @DataStore\libs\AceTimer-3.0\AceTimer-3.0.lua:94
 DisableModule = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:381
 ADDON_LOADED = <function> defined @SavedInstances\SavedInstances.lua:2502
 RegisterMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 UnregisterMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 RegisterBucketEvent = <function> defined @SavedInstances\Libs\AceBucket-3.0\AceBucket-3.0.lua:184
 memcheck = <function> defined @SavedInstances\SavedInstances.lua:2868
 scan_item_cds = <function> defined @SavedInstances\SavedInstances.lua:4269
 localarr#showcol = <table> {
 }
 SetEnabledState = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:455
 enabledState = true
 RegisterEvent = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 localarr#RefreshTemp = <table> {
 }
 IterateModules = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:468
 TradeSkillRescan = <function> defined @SavedInstances\SavedInstances.lua:4355
 CHAT_MSG_MONSTER_YELL = <function> defined @SavedInstances\SavedInstances.lua:2593
 localarr#wbsave = <table> {
 }
 OnInitialize = <function> defined @SavedInstances\SavedInstances.lua:2351
 NewModule = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:266
 GetModule = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:241
 ShowTooltip = <function> defined @SavedInstances\SavedInstances.lua:3337
 defaultModuleLibraries = <table> {
 }
 TimeLeft = <function> defined @DataStore\libs\AceTimer-3.0\AceTimer-3.0.lua:172
 SendMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:59
 ScheduleRepeatingTimer = <function> defined @DataStore\libs\AceTimer-3.0\AceTimer-3.0.lua:129
 CancelAllTimers = <function> defined @DataStore\libs\AceTimer-3.0\AceTimer-3.0.lua:160
 toonInit = <function> defined @SavedInstances\SavedInstances.lua:2335
 localarr#instancerow = <table> {
 }
 GetName = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:310
 QuestRefresh = <function> defined @SavedInstances\SavedInstances.lua:2880
 localarr#lfrbox = <table> {
 }
 localarr#instancesaved = <table> {
 }
 name = "SavedInstances_Core"
 localarr#categoryshown = <table> {
 }
 RegisterBucketMessage = <function> defined @SavedInstances\Libs\AceBucket-3.0\AceBucket-3.0.lua:200
 orderedModules = <table> {
 }
 scale = 1
 Disable = <function> defined @Accountant\Libs\AceAddon-3.0\AceAddon-3.0.lua:345
 Enable = <function> defined @Accoun

**Comment by LiangYuxuan** [comment]
No idea about this, is it still happening after reloading ui?

**Comment by raysmith59** [comment]
Yes, it's happening every time on every toon.  Basically as soon as my cursor hits the key icon the error occurs.

![image](https://user-images.githubusercontent.com/2917594/51052243-17d27880-15a4-11e9-80c9-9b9351aaad29.png)

**Comment by LiangYuxuan** [comment]
Could you try reinstall the addon?
